### PR TITLE
Issue 529 add

### PR DESCRIPTION
**Pull request by krushiraj**
# Added feedback form

### Checklist for this PR

* [x] I have read [contributing guidelines](https://github.com/coderplex/coderplex/blob/develop/.github/CONTRIBUTING.md).
* [x] I executed `yarn lint` locally to format my code.
* [ ] I added relevant Tests to verify my code. (N/A)
* [x] All tests `yarn test` ran successfully.
* [x] Pull request is towards `develop` branch.
* [x] I have tested the responsiveness of the UI by checking on devices using [ScreenFly](http://quirktools.com/screenfly/) 
* [x] Added myself to contributors table 
* [ ] Documentation
* [x] Ready to be merged

### CODE REVIEW

File: pages/feedback.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import Form from '../components/common/airtable-form';
+
+console.log(Form);
```
Comment (vinaypuppal) [inline pages/feedback.js]: remove this log

**Comment by vinaypuppal** [comment]
#### 📝Changed routes:


| [**/feedback**](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/feedback) | [**/join**](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/join) |
|:-:|:-:|
|<a href="https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/feedback">Screenshot of /feedback</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/A1uCkKs_Aln89n/JRFrFRw1GsH1Q_">(view full size)</a> |<a href="https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/join">Screenshot of /join</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/hPNylvD_VG6wKq/ni5ew6TYAFrYI_">(view full size)</a> |







Commit 57e11442a552a6b71989ead0e8cea98e41d116a3 (https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh).

**Comment by krushiraj** [comment]
> #### 📝Changed routes:
> [**/feedback**](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/feedback)	[**/join**](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/join)
> [Screenshot of /feedback](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/feedback)
> [(view full size)](https://microlink-cdn.s3.amazonaws.com/s/A1uCkKs_Aln89n/JRFrFRw1GsH1Q_)	[Screenshot of /join](https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh/join)
> [(view full size)](https://microlink-cdn.s3.amazonaws.com/s/hPNylvD_VG6wKq/ni5ew6TYAFrYI_)
> Commit [57e1144](https://github.com/coderplex/coderplex/commit/57e11442a552a6b71989ead0e8cea98e41d116a3) (https://coderplex-git-fork-krushiraj-issue-529-add.coderplex.now.sh).

Sorry I didn't get this.

**Comment by vinaypuppal** [comment]
@krushiraj Nevermind about it. It's auto-generated by now.sh bot. You can read more about it here https://zeit.co/blog/deploy-summary

**Comment by vinaypuppal** [comment]
Mobile nav seems to be broken, I think we need to fix the `top` value which hides the mobile nav. Please check and fix it.

Screenshot 2019-09-06 at 5 52 30 PM

**Pull request merged**